vendor/k8s.io/client-go/pkg/version/base.go

```diff
@@ -55,8 +55,8 @@ var (
 	// NOTE: The $Format strings are replaced during 'git archive' thanks to the
 	// companion .gitattributes file containing 'export-subst' in this same
 	// directory.  See also https://git-scm.com/docs/gitattributes
-	gitVersion   string = "v0.0.0-master+$Format:%h$"
-	gitCommit    string = "$Format:%H$" // sha1 from git, output of $(git rev-parse HEAD)
+	gitVersion   string = "v0.0.0-master+7d04d0e2"
```

Comment:
This doesn't look right.

Reply:
It looks like client-go updated the way they generate version strings? What looks off to you?

Reply:
I don't think it has changed? (Not sure which is the source of truth, but they are equal.)
https://github.com/kubernetes/client-go/blob/master/pkg/version/base.go#L58
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/pkg/version/base.go#L58

Reply:
are we seeing the result of:

> // NOTE: The $Format strings are replaced during 'git archive' thanks to the 
> // companion .gitattributes file containing 'export-subst' in this same
> // directory.  See also https://git-scm.com/docs/gitattributes

I wonder if k8s.io changed to start serving the git archive instead of the git repo?

Reply:
Maybe so, will assume that's correct.

Reply:
Nick's PR actually undoes the change in operator-registry, so maybe it was a mistake after all. - https://github.com/operator-framework/operator-registry/pull/19.